index.html

```diff
@@ -1,37 +1,58 @@
 <!DOCTYPE html>
-<html>
```

Comment:
shouldn't be removed, the doctype tag is not the same as the html tag. I'm guessing this is the problem with prettier. try to add it back and return prettier